Modules/Quotas/src/Quotas.Application/Tiers/Commands/SeedQueuedForDeletionTier/Handler.cs

```diff
@@ -27,7 +27,7 @@ public async Task Handle(SeedQueuedForDeletionTierCommand request, CancellationT
         }
 
         var metrics = await _metricsRepository.FindAll(CancellationToken.None);
-        queuedForDeletionTier.AddQuotaForAllMetricsOnQueuedForDeletion(metrics);
+        queuedForDeletionTier.AddQuotaForAllMetricsOnQueuedForDeletion(metrics.Where(m => m.Key != MetricKey.NumberOfCreatedDatawalletModifications && m.Key != MetricKey.NumberOfStartedDeletionProcesses));
```

Comment:
You should extract the filtered metrics into a variable and add a comment explaining why we exclude the two metrics.